static/usr/lib/systemd/system/remount-core-fs.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Reset propagation of initial mount points
+DefaultDependencies=no
+Before=local-fs-pre.target
+Before=local-fs.target
```

Comment:
Is this needed? `local-fs-pre.target` is expected to happen before afaiu.

Reply:
This is in case something happens with `local-fs-pre.target`, like something making it fail.

Note that I used roughly the same dependencies as `systemd-remount-fs.service`.